src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1136,22 +1136,16 @@ static void uct_dc_mlx5_ep_fc_cleanup(uct_dc_mlx5_ep_t *ep)
     }
 }
 
-UCS_CLASS_CLEANUP_FUNC(uct_dc_mlx5_ep_t)
+UCS_CLASS_CLEANUP_FUNC(uct_dc_mlx5_ep_t)    
 {
     uct_dc_mlx5_iface_t *iface = ucs_derived_of(self->super.super.iface,
                                                 uct_dc_mlx5_iface_t);
-    khiter_t it;
 
     uct_dc_mlx5_ep_pending_purge(&self->super.super,
                                  uct_rc_ep_pending_purge_warn_cb, self);
     uct_dc_mlx5_ep_fc_cleanup(self);
     uct_dc_mlx5_ep_keepalive_cleanup(self);
 
-    it = kh_get(uct_dc_mlx5_fc_hash, &iface->tx.fc_hash, (uint64_t)self);
```

Comment:
can we just move `uct_rc_fc_cleanup(&ep->fc);` here and remove `uct_dc_mlx5_ep_fc_cleanup` instead? This fc cleanup func is very small and is called from one place only.

Reply:
@evgeny-leksikov @hoopoepg wdyt?
I prefer having `uct_dc_mlx5_ep_fc_cleanup` to keep all FC-related things in a single place.
actually, `uct_rc_fc_cleanup` is 1-line function, but we still have it

Reply:
I like current @dmitrygx's proposal - less code in destructor

Reply:
👍 to keep all FC-related things in a single place